data/json/recipes/recipe_others.json

```diff
@@ -3669,6 +3669,32 @@
     "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ], [ [ "forge", 50 ], [ "oxy_torch", 10 ] ] ],
     "components": [ [ [ "scrap", 3 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "railroad_rail_small",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "time": "1 h",
+    "autolearn": true,
+    "using": [ [ "steel_standard", 10 ] ],
```

Comment:
As we discussed on discord, this makes more sense as steel-plated wood rails rather than a forged rail, which seems like an immense amount of work with an anvil and forge.

Reply:
merged railroad rail into the railroad track recipe and also decreased the steel requirement and increased the wood requirement